cmd/shortener/main.go

```diff
@@ -3,18 +3,22 @@ package main
 import (
 	"github.com/avidyakov/shortener/internal/config"
 	"github.com/avidyakov/shortener/internal/handlers"
+	"github.com/avidyakov/shortener/internal/logger"
 	"github.com/avidyakov/shortener/internal/repositories"
-	"log"
+	"go.uber.org/zap"
 	"net/http"
 )
 
 func main() {
-	log.Printf("Initializing config")
-	cfg := config.NewConfig()
+	logger.Log, _ = zap.NewProduction()
+	defer logger.Log.Sync()
 
-	log.Printf("Initializing handlers")
-	handler := handlers.NewLinkHandlers(repositories.NewMemoryLink(), cfg.BaseURL)
+	logger.Log.Info("Initializing server configuration and handlers")
+	conf := config.NewConfig()
+	handler := handlers.NewLinkHandlers(repositories.NewFileRepo(conf.File), conf.BaseURL)
```

Comment:
Сложно читать, особенно если увеличится количество необходимых параметров
```suggestion
       rep:= repositories.NewFileRepo(conf.File)
	handler := handlers.NewLinkHandlers(rep), conf.BaseURL)
```